selenium/src/test/java/au/org/emii/portal/tests/BrowserStackTest.java

```diff
@@ -0,0 +1,19 @@
+package au.org.emii.portal.tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class BrowserStackTest extends BaseTest {
+
+    @Test(invocationCount = invocationCount, threadPoolSize = threadPoolSize)
```

Comment:
Perhaps only use invocationcount and threadPoolSize were the test require it (none of yours do so I'd purge these references from the project altogether)..